src/Host/Broker/Impl/Sessions/SessionManager.cs

```diff
@@ -68,7 +69,8 @@ public class SessionManager {
                 password,
                 profilePath,
                 _loggingOptions.LogHostOutput ? _hostOutputLogger : null,
-                _loggingOptions.LogPackets ? _messageLogger : null);
+                _loggingOptions.LogPackets ? _messageLogger : null,
+                _loggingOptions.LogPackets ? LogVerbosity.Traffic : LogVerbosity.Minimal);
```

Comment:
There are 3 level and when logging something code can specify which min verbosity the message appears at. This is different from general concept of verbosity as in msbuild, for example. Regular concept is trace -> detailed verbosity, warning -> regular, error -> minimal. However, we also need to take into account privacy and security so level/severity is different from verbosity. 'Traffic' means that log entry may contain traffic data and if traffic logging is off, traffic-related errors won't get logged.